psi-ms.obo

```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
 relationship: has_column: UO:0000191 ! fraction
 relationship: has_relation MS:1000285 ! total ion current
 
+[Term]
+id: MS:4000069
+name: m/z acquisition range
+def: "Upper and lower limit of m/z precursor values at which MSn spectra are recorded." [PSI:QC]
```

Comment:
This seems to refer to traditional DDA MS/MS spectra? For DIA or BoxCar acquisition modes, the m/z range is relevant for MS1 scans as well, and will take different values. I don't think this term needs to change, I just wanted to briefly highlight these related use cases.

Reply:
Should we add a comment there?